src/SearchResults.js

```diff
@@ -1,35 +1,79 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+import { numberOfNight } from "./Nights";
+
+const SearchResults = ({ results }) => {
+  const [selectedRow, setSelectedRow] = useState(null);
+  const [customerData, setCustomerData] = useState(null);
+
+  useEffect(() => {
+    if (selectedRow) {
+      fetch(`https://cyf-react.glitch.me/customers/${selectedRow}`)
+        .then((response) => response.json())
+        .then((data) => setCustomerData(data));
```

Comment:
make sure to add a `.catch` as this network request could fail, and you'll want some way of letting the user know that the request has failed